frontend/src/scenes/session-recordings/player/rrweb/canvas/canvas-plugin.ts

```diff
@@ -220,8 +220,7 @@ export const CanvasReplayerPlugin = (events: eventWithTime[]): ReplayPlugin => {
             }
         },
 
-        // eslint-disable-next-line @typescript-eslint/no-misused-promises
-        handler: async (e: eventWithTime, isSync: boolean, { replayer }: { replayer: Replayer }) => {
+        handler: (e: eventWithTime, isSync: boolean, { replayer }: { replayer: Replayer }) => {
```

Comment:
this doesn't need to be returning a promise

it is possible that we're leaking the reference to the replayer into a promise that is never handled and that the replayer then keeps everything alive

i did see a reference to the canvas plugin but my browser tab has now crashed and I can't reference it